Assets/MRTK/package.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "com.microsoft.mixedreality.toolkit.foundation",
+    "displayName": "Mixed Reality Toolkit Foundation",
+    "version": "2.5.0-test.1",
```

This one didn't match the others:

```suggestion
    "version": "2.5.0-preview.0",
```